Week-6/assignment5.js

```diff
@@ -0,0 +1,15 @@
+// Pair With Given Difference
```

Comment:
i believe the question is to find a pair with the difference closest to the number. 
Also, please add problem statements to all the questions as a comment along with time and space complexity

Reply:
Added Problem statement and Time complexity in the comment
Also , It says we have to find whether the pair exist or not and return 1|0 accordingly.
Closest target is the 6.6 problem
![image](https://user-images.githubusercontent.com/54816661/187711953-87766c60-5ac6-49af-8c06-05f2e126f65d.png)